studies/complexity_structure/analysis.Rmd

```diff
@@ -5,15 +5,16 @@ editor_options:
 
 There has been an exponential increase in the number of complexity indices in the past few decades [@yang2013]. Although these new procedures are usually mathematically well-defined and theoretically promising, limited empirical evidence is available to understand their similarities and differences [@lau2021brain; @yang2013]. Moreover, some of these methods are resource-intensive and require long computation times. This complicates their application with techniques that utilise high sampling-rates (e.g., M/EEG) and makes them impractical to implement in real-time settings - such as brain-computer interfaces [@azami2017refined; @manis2018]. As such, having empirical data about the computation time of various complexity indices would prove useful, for instance to objectively guide their selection, especially in contexts where time or computational resources are limited.
 
-Additionally, the lack of a comprehensive open-source and user-friendly software for computing various complexity indices likely contributes to the limited availability of empirical comparison [@flood2021]. Indeed, most complexity indices are only described mathematically in journal articles, with reusable code seldom made available, therefore limiting their further application and validation [@yang2013; @flood2021]. To address this gap, we added a comprehensive set of complexity-related features to *NeuroKit2*, a Python package for physiological signal processing [@Makowski2021neurokit], to provide users with a software to compute a vast amount of complexity indices. The code is designed to be as fast as possible, while still written in pure Python [though with the help of dependencies such as Numpy or Pandas, @harris2020array; @mckinney2010data] to maximize the re-usability, transparency, and correctness.
+Additionally, the lack of a comprehensive open-source and user-friendly software for computing various complexity indices likely contributes to the limited availability of empirical comparison [@flood2021]. Indeed, many complexity indices are only described mathematically in journal articles, with reusable code seldom made available, therefore limiting their further application and validation [@yang2013; @flood2021]. And complexity measures with open-source code implementations are scattered across different packages or scripts, or embedded within a larger goal-directed framework [e.g., *HCTSA*, a time-series comparison tool, @fulcher2017]. To address this lack of unified accessibility, we added a comprehensive set of complexity-related features to *NeuroKit2*, a Python package for physiological signal processing [@Makowski2021neurokit], to provide users with a software to compute a vast amount of complexity indices. The code is designed to be as fast as possible, while still written in pure Python [with the help of standard dependencies such as *NumPy* or *Pandas*, @harris2020array; @mckinney2010data] to maximize the re-usability, transparency, and correctness.
```

Comment:
@anshu-97 yes I started the sentence with "And" but I think in this particular context it works in english lol

Reply:
@DominiqueMakowski ?!?!😫 Can it be rephrased to "Even when made accessible on open source platforms, implementations of their code are often found scattered...."^^